pabi_procurement/models/purchase_request.py

```diff
@@ -85,26 +96,59 @@ class PurchaseRequest(models.Model):
     @api.model
     def create(self, vals):
         create_rec = super(PurchaseRequest, self).create(vals)
```

Comment:
If you don't know what variable name should be. You can always use **res** to get the result. :)